jsonschema-hyperschema.xml

```diff
@@ -2248,43 +2278,32 @@ Link: <https://api.example.com/trees/1/nodes/456> rev=up
-                    </postamble>
-                </figure>
-                <t>
-                    <cref>
-                        It's not clear how pagination should work with the link from the
-                        entry point resource or the "collection" link in the item.
-                        Technically, link from an item to a paginated or filtered
-                        collection should go to a page/filter that contains the link
-                        context.  If the collection is using semantic sorting instead
-                        of a plain integer limit and offset, then this is possible.
-                        But getting into semantic pagination seems way too involved for this
-                        example.
-                    </cref>
-                </t>
-                <t>
-                    <cref>
-                        Note also that POST-ing to a collection page that will not contain
-                        the created item also seems weird.  While retrieving the collection
-                        from a query parameter-less URI will still retrieve a page, that's
-                        a limit imposed by the server.  POST-ing to such a URI and disregarding
-                        the "default" values for the parameters seems correct.  This is another
-                        reason to *not* automatically write default values into the query.
-                    </cref>
-                </t>
-                <t>
-                    Let's also update our entry point link to include pagination, specifically
-                    allowing arbitrary input to choose the page:
-                </t>
-                <figure>
-                    <artwork>
+                        </artwork>
+                        <postamble>
+                            Note that there is no "prev" link in the output, as we are looking
+                            at the first page.  The lack of a "prev" field under "meta",
+                            together with the "prev" link's "hrefRequired" values, means
+                            that the link is not usable with this particular instance.
+                        </postamble>
+                    </figure>
+                    <t>
+                        <cref>
+                            It's not clear how pagination should work with the link from
+                            the "collection" links in the individual "thing" schema.
+                            Technically, a link from an item to a paginated or filtered
+                            collection should go to a page/filter that contains the item
+                            (in this case the "thing") that is the link context.
+                            See GitHub issue #421 for more discussion.
+                        </cref>
+                    </t>
+                    <t>
+                        Let's add a link for this collection to the entry point schema, including
```

Comment:
Might be worth adding a reference to the "entry point schema" which is currently defined in section 9.1 (we are in section 9.5.1).

Reply:
Good call.